src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,105 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double SIZE_MULTIPLIER = 1.5;
+    private static final String WRONG_INDEX = "Index is out of list size";
```

Comment:
I think we can just use message "Index is out of list size" in place we need it (it is really hard to read code when it moved to constant)